OrbitGl/ThreadTrack.cpp

```diff
@@ -262,6 +265,10 @@ void ThreadTrack::OnTimer(const Timer& timer) {
   if (timer.m_End > max_time_) max_time_ = timer.m_End;
 }
 
+std::string ThreadTrack::GetTooltip() const {
+  return "Shows collected samples and instrumented function timings (if existing).";
```

Comment:
I would probably get rid of "if existing", it is implied anyway. WDYT?

Reply:
Maybe an idea: If you write "...and timings from dynamically instrumented functions." it is a user hint that one can dynamically instrument functions, which can aid when learning and discovering the tool.

Reply:
I initially added the "if existing" because it might have been confusing if the user does not know about dynamic instrumentation and would be looking for the timings announced in the tooltip, but I think yours is more elegant and also implies this. Changed.